### PR TITLE
Android Intent Command Control: Added extra shows plus instead of minus

### PR DESCRIPTION
**Pull request by ifarzana**
Fixes: https://github.com/xibosignage/xibo/issues/3023

### CODE REVIEW

File: views/authed.twig [hunk 1]
```diff
@@ -306,7 +306,7 @@
                 <option value="intArray" {{#eq type "intArray"}}selected="selected"{{/eq}}>intArray</option
             </select>
             <input class="form-control form-control-inline validated-element extra-value" placeholder="{% endverbatim %}{% trans "Value" %}{% verbatim %}" value="{{ value }}">
-            <button type="button" class="btn btn-sm btn-warning pull-right intent-remove-extra"><i class="fa fa-minus fa-plus"></i></button>
+            <button type="button" class="btn btn-sm btn-warning pull-right intent-remove-extra"><i class="fa fa-minus"></i></button>
```
Comment (maurofmferrao) [inline views/authed.twig:309]: This one works, but I think you've missed the same fix in line 400 for the `command-input-http-key-value-template` template

**Review by maurofmferrao — CHANGES_REQUESTED** [review]
We just need the same fix for `command-input-http-key-value-template` :)

File: views/authed.twig (hunk 1 above)
Comment (ifarzana) [reply views/authed.twig:309]: Thanks for pointing that

**Review by maurofmferrao — APPROVED** [review]
Looks good to me!

**Pull request merged**